### PR TITLE
Add new Linters content

### PR DESCRIPTION
**Pull request by samgnaniah**
Adds new content on Linters.

### CODE REVIEW

**Review by chanikag — APPROVED** [review]

File: en/docs/design/advanced-topics/validate-api-definitions-with-linters.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Validating API Definitions Against Custom Linter Rules
+
+By default, API Definitions are validated against default OpenAPI schema when importing and writing or editing them inside the publisher portal. Apart from the default OpenAPI validation,  the APIs created by the API publisher can be validated against a set of custom rules and it is useful in standardizing and keeping API definitions clean.
```
Comment (tgtshanika) [inline en/docs/design/advanced-topics/validate-api-definitions-with-linters.md:3]: Additional space after comma-> OpenAPI validation,  the

File: en/docs/design/advanced-topics/validate-api-definitions-with-linters.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Now, the new rule is enforced and the API Publisher can observe the validation results in three scenarios.
+
+### When creating an API from OpenAPI definition
+
+When you are following the [API creation steps]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api-from-an-openapi-definition), the linter results will be shown as below.
+
+[![Linter results]({{base_path}}/assets/img/design/create-api/linter-observe/linters2.png)]({{base_path}}/assets/img/design/create-api/linter-observe/linters2.png)
+
+!!! Note 
+    Currently, the linter results feature is supported only when `OpenAPI URL` and `OpenAPI File` input types are used.
```
[the remark this answers is not in the document]
Comment (samgnaniah) [reply en/docs/design/advanced-topics/validate-api-definitions-with-linters.md]: Fixed with https://github.com/wso2/docs-apim/pull/6531/commits/d4f73e17e615acdef46fba31312245a0fe4a3a39

File: en/docs/design/advanced-topics/validate-api-definitions-with-linters.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/design/advanced-topics/validate-api-definitions-with-linters.md]: Fixed with https://github.com/wso2/docs-apim/pull/6531/commits/d4f73e17e615acdef46fba31312245a0fe4a3a39

[the remark this answers is not in the document]
Comment (samgnaniah) [reply en/docs/design/advanced-topics/validate-api-definitions-with-linters.md]: Fixed with https://github.com/wso2/docs-apim/pull/6531/commits/d4f73e17e615acdef46fba31312245a0fe4a3a39

**Pull request merged**